src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -961,15 +967,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <Target Name="BuildGenerateSources" DependsOnTargets="BuildGenerateSourcesTraverse;$(BuildGenerateSourcesAction)" />
 
-  <Target Name="BuildGenerateSourcesTraverse" DependsOnTargets="PrepareProjectReferences">
+  <Target Name="BuildGenerateSourcesTraverse" DependsOnTargets="PrepareProjectReferences;AddOutputPathToGlobalPropertiesToRemove">
```

Comment:
Could you fold the new target into `PrepareProjectReferences`? Either directly or by hooking before/after it?

Reply:
Do _GetProjectReferenceTargetFrameworkProperties and _GetCopyToOutputDirectoryItemsFromTransitiveProjectReferences require PrepareProjectReferences?